python/rrc_simulation/trifinger_platform.py

```diff
@@ -386,9 +384,9 @@ def store_action_log(self, filename):
         t = self.simfinger.get_current_timeindex()
         object_pose = self.get_object_pose(t)
         self._action_log["final_object_pose"] = {
-            "position": object_pose.position.tolist(),
-            "orientation": object_pose.orientation.tolist(),
+            "t": t,
```

Comment:
minor suggestion: would it make sense to call this "final_object_pose_stamped" or something like that? otherwise i found it a bit confusing that one has to call log["final_object_pose"]["pose"]